src/components/Swap/Settings/index.tsx

```diff
@@ -1,72 +1,48 @@
-import { Trans } from '@lingui/macro'
+import { useCloseOnEscape } from 'hooks/useCloseOnEscape'
 import { Settings as SettingsIcon } from 'icons'
-import { useAtomValue, useResetAtom } from 'jotai/utils'
-import { useCallback, useState } from 'react'
-import { swapEventHandlersAtom } from 'state/swap'
-import { settingsAtom } from 'state/swap/settings'
+import { useState } from 'react'
 import styled from 'styled-components/macro'
-import { ThemedText } from 'theme'
 
-import { IconButton, TextButton } from '../../Button'
+import { IconButton } from '../../Button'
 import Column from '../../Column'
-import Dialog, { Header } from '../../Dialog'
-import { BoundaryProvider } from '../../Popover'
+import Popover, { PopoverBoundaryProvider } from '../../Popover'
 import MaxSlippageSelect from './MaxSlippageSelect'
 import TransactionTtlInput from './TransactionTtlInput'
 
-export function SettingsDialog() {
+export function SettingsPopover() {
   const [boundary, setBoundary] = useState<HTMLDivElement | null>(null)
-  const { onSettingsReset } = useAtomValue(swapEventHandlersAtom)
-  const resetSettingsBase = useResetAtom(settingsAtom)
-  const resetSettings = useCallback(() => {
-    onSettingsReset?.()
-    resetSettingsBase()
-  }, [onSettingsReset, resetSettingsBase])
+
+  // TODO: add back reset settings functionality
```

Comment:
tag the todo with your name or (preferably) the ticket ID (eg WEB-123). This way, if it is forgotten, it will be easy to look up the corresponding ticket in the future.